acousticbrainz/views/api.py

```diff
@@ -7,11 +7,32 @@
 api_bp = Blueprint('api', __name__)
 
 
+@api_bp.route("/<uuid:mbid>/meta", methods=["GET"])
+@crossdomain()
+def meta(mbid):
+    return jsonify({
+        'mbid': mbid,
+        'count': count_lowlevel(mbid),
+    })
+
+
 @api_bp.route("/<uuid:mbid>/low-level", methods=["GET"])
 @crossdomain()
 def get_low_level(mbid):
-    """Endpoint for fetching low-level information to AcousticBrainz."""
-    return Response(load_low_level(mbid), content_type='application/json')
+    """Endpoint for fetching low-level information to AcousticBrainz.
+
+    Offset can be specified if you need to get another dataset in case there
+    are duplicates.
+    """
+    offset = request.args.get("n")
+    if offset:
```

Comment:
It is a number. But rather than describing the type, I think we should describe the content...